Puppetfile

```diff
@@ -39,21 +39,21 @@ github "module_data", "0.0.4", :repo => "ripienaar/puppet-module-data"
 # some/most of these if you want, but it's not recommended.
 
 github "brewcask",    "0.0.10"
-github "dnsmasq",     "2.0.3"
+github "dnsmasq",     "2.0.4"
 github "foreman",     "1.2.0"
-github "gcc",         "3.0.2"
+# mod "gcc", "0.0.1", :git => "https://github.com/boxen/puppet-gcc.git", :ref => "abc7639"
```

Comment:
yep, this is gonna stay commented out until i can see if this actually breaks anything